src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/Link.js

```diff
@@ -143,11 +146,11 @@ class Link extends Component<Props> {
         );
 
         const allowedTypes = linkTypeRegistry.getKeys().filter((key) => {
-            if (types === undefined || types.length === 0) {
-                return true;
+            if (types !== undefined && types.length > 0) {
+                return types.includes(key);
```

Comment:
i think i would expect that both, `types` and `excludedTypes` are considered if both are set.
so if both are set, we should only display the types that are inside of `types` bot not inside of `excludedTypes`.